assets/js/slideNav.js

```diff
@@ -0,0 +1,68 @@
+/* jshint esversion: 6 */
+
+//TODO: Create perm open attr.
```

Comment:
Most of these are done. Will be cleaned up after prototype approved before merge request.